ecl_type_traits/include/ecl/type_traits/numeric_limits.hpp

```diff
@@ -172,36 +170,36 @@ class ecl_type_traits_PUBLIC numeric_limits<unsigned int> : public std::numeric_
 
 
 /**
- * @brief Expands the std numeric_limits class for long.
+ * @brief Expands the std numeric_limits class for ecl:int64.
  *
  * @sa ecl::numeric_limits
  */
 template <>
-class ecl_type_traits_PUBLIC numeric_limits<long> : public std::numeric_limits<long> {
+class ecl_type_traits_PUBLIC numeric_limits<int64> : public std::numeric_limits<long> {
 public:
-	static const long one = 1L;
+	static const int64 one = 1L;
 	static const uint16 bits = ECL_SIZE_OF_LONG*8;
 	static const uint16 bytes = ECL_SIZE_OF_LONG;
-	static const long minimum = LONG_MIN;
-	static const long maximum = LONG_MAX;
+	static const int64 minimum = LONG_MIN;
+	static const int64 maximum = LONG_MAX;
 
 	typedef float Precision;
 	static const Precision dummy_precision;
 };
 
 /**
- * @brief Expands the std numeric_limits class for unsigned long.
+ * @brief Expands the std numeric_limits class for ecl::uint64.
  *
  * @sa ecl::numeric_limits
  */
 template <>
-class ecl_type_traits_PUBLIC numeric_limits<unsigned long> : public std::numeric_limits<unsigned long> {
+class ecl_type_traits_PUBLIC numeric_limits<uint64> : public std::numeric_limits<unsigned long> {
```

Comment:
Not sure where do `uint64` get defined but `uint64_t` is a more standards-conforming type.

Reply:
the `uint64` here is in `ecl` namespace, defined in https://github.com/ms-iot/ecl_lite/blob/windows_bringup/ecl_config/include/ecl/config/portable_types.hpp

Reply:
Got it. Now it makes sense.